### DISCUSSION TITLE
Difference between Total Power and individual circuit power

### DISCUSSION THREAD

**Discussion by Roemer** [emporia-vue-local/esphome#129] [Q&A]
There is a small difference between the total power entity and the individual power entity.
In Home Assistant they look like:
Total:
```
unit_of_measurement: W
friendly_name: Total Power
```
Individual:
```
state_class: measurement
unit_of_measurement: W
device_class: power
friendly_name: Circuit 10 Power
```
This leads for example to different exports to prometheus:
Total:
```
hass_sensor_unit_w{domain="sensor",entity="sensor.total_power",friendly_name="Total Power"} 3188.5
```
Individual:
```
hass_sensor_power_w{domain="sensor",entity="sensor.circuit_1_power",friendly_name="Circuit 1 Power"} 42.8
```

I know, this is nitpicking but I think they should be the same?

**Comment by flaviut** [comment]
This is something to do with "device_class" or "state_class". If you have a few moments to figure this out, I'd appreciate an PR to the config in the readme.
